Makefile

```diff
@@ -1,7 +1,7 @@
-IMAGE_VERSION := $(shell python3 scripts/retrieve_version.py)
+IMAGE_VERSION := $(shell python scripts/retrieve_version.py)
 SHORTENED_LATEST_VERSION := $(shell echo $(IMAGE_VERSION) | awk -F. '{print $$1"."$$2}')
 KOGITO_APPS_TARGET_BRANCH ?= main
-KOGITO_APPS_TARGET_URI ?= https://github.com/kiegroup/kogito-apps.git
+KOGITO_APPS_TARGET_URI ?= https://github.com/apache/incubator-kie-kogito-apps.git
```

Comment:
We will probably have to create a script to replace these values and have `Makefile` version for prod.